contracts/interfaces/0.8.x/IAdminACLV0.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+// Created By: Art Blocks Inc.
+
+pragma solidity ^0.8.0;
+
+interface IAdminACLV0 {
+    // Type of the Admin ACL contract of the form "AdminACLV0"
+    function AdminACLType() external view returns (string memory);
+
+    // Checks if sender is allowed to call function with selector `_selector`
+    function allowed(address _sender, bytes4 _selector) external returns (bool);
```

Comment:
I think we want this method to also take a smart contract address, for the address of contract that the `_selector` is associated with–unless intention here is to use `msg.sender` in which case we should make this explicitly documented in the interface (though personally I prefer just sending address along explicitly vs. relying on `msg.sender`).

Reply:
I have a preference of using `msg.sender` here since it is the true source. e.g. you would never be able to accidentally have a typo in a core contract.

That being said, I can see a use case of a contract wanting to check if a sender has approval on some contract other than itself. How do you feel about overloading `allowed` and adding another function to the interface:
```
function allowed(address _sender, bytes4 _selector, address _contract) external returns (bool);
```
... and updating documentation like you suggest?

Reply:
I don't think the typo reason is a good reason, as the same could impact the method signature and that part is passed explicitly. 

The overloading approach sounds good to me if we think we do really want to use the `msg.sender` approach, but I am quite reluctant as I just really don't love how non-explicit it is. 

Even with proper documentation it feels a bit obfuscated (taking the philosophy here that methods should ideally be self-documenting, not requiring reading of the implicit assumptions in dev documentation). 

How compelling do we think the benefit of using `msg.sender` is (may be worth some synchronous convo here, I'm a bit torn).

Reply:
hmm, yeah chewing on this more, from the perspective of "ideally self documenting", I'm good with adding an additional arg here instead of using `msg.sender`. If I dig deeper into my psyche here, my initial preference for `msg.sender` was probably driven by gas efficiency more than anything else, but realistically, these functions are not in our mint sequence, and readability/conciseness is more important than <1% gas savings.

Thanks for chewing on this one, will update shortly!

Reply:
Yeah I think if this intersected with the minting sequence at all I would definitely feel differently here, but given that these are just for administration method calls, and will have a very low % impact on that (we should definitely still be mindful of _high gas impact_ changes to administration methods) would definitely prefer to optimize for readability in favor of a slight gas trade off. SGTM ser 🙏 TY for the back and forth.

Reply:
(updated)

Reply:
LGTM 🙌  🙌  🙌